sql/core/src/main/scala/org/apache/spark/sql/streaming/DataStreamWriter.scala

```diff
@@ -288,19 +289,21 @@ final class DataStreamWriter[T] private[sql](ds: Dataset[T]) {
        * Note, currently the new table creation by this API doesn't fully cover the V2 table.
        * TODO (SPARK-33638): Full support of v2 table creation
        */
-      val cmd = CreateTableStatement(
-        originalMultipartIdentifier,
+      val cmd = CreateV2Table(
+        UnresolvedDBObjectName(
+          originalMultipartIdentifier,
+          isNamespace = true),
         df.schema.asNullable,
         partitioningColumns.getOrElse(Nil).asTransforms.toSeq,
         None,
         Map.empty[String, String],
-        Some(source),
         Map.empty[String, String],
-        extraOptions.get("path"),
         None,
+        extraOptions.get("path"),
         None,
+        Some(source),
         external = false,
-        ifNotExists = false)
+        ignoreIfExists = false)
```

Comment:
Just for my information, is `ignoreIfExists` like `ifNotExists` of `CreateTableStatement`? Thanks!

Reply:
Yes. We create table if the table doesn't exist, it's the same as we ignore if the table exists.